.github/workflows/build.yml

```diff
@@ -0,0 +1,27 @@
+name: build
+
+on:
+  release:
+    types: [released]
+  pull_request:
+  workflow_dispatch:
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  build:
+    uses: OpenAstronomy/github-actions-workflows/.github/workflows/publish.yml@v1
+    with:
+      upload_to_pypi: ${{ (github.event_name == 'release') && (github.event.action == 'released') }}
+      targets: |
+        # Linux wheels
+        - cp3*-manylinux_x86_64
+        # MacOS wheels
+        - cp3*-macosx_x86_64
+        # Until we have arm64 runners, we can't automatically test arm64 wheels
+        - cp3*-macosx_arm64
+      sdist: true
+    secrets:
+      pypi_token: ${{ secrets.PYPI_PASSWORD_STSCI_MAINTAINER }}
```

Comment:
I am not sure if you can reuse the old password in this workflow. Do you know for sure this will work?

Reply:
yes indeed, the `PYPI_PASSWORD_STSCI_MAINTAINER` secret is actually an API token for the PyPI account (I dunno why they named it that)